sync/test_sync.py

```diff
@@ -206,13 +181,13 @@ def test_transform_text(self):
         tmp_name = None
 
         # write to file
-        with tempfile.NamedTemporaryFile(delete=False) as tmp:
+        with tempfile.NamedTemporaryFile(dir='/tmp', delete=False) as tmp:
```

Comment:
Assuming `/tmp` is the temp folder is not portable, so I set the base dir explicitly to make the test run on mac.